reference/5.1/Microsoft.PowerShell.Core/About/about_numeric_literals.md

```diff
@@ -0,0 +1,190 @@
+  then its type is `[long]`.
+- If its value can be represented by type specified then that is its type.
+- Otherwise, that literal is ill formed.
+
+## Real literals
+
+Real literals can only be written in decimal notation. This notation can
+include fractional values following a decimal point and scientific notation
+using an exponential part.
+
+The exponential part includes an 'e' followed by an optional sign (+/-) and a
+number representing the exponent. For example, then literal value `1e2` equals
+the numeric value 100.
+
+Real literals can have a type suffix and a multiplier suffix.
+
+| Suffix |       Meaning       |
+| ------ | ------------------- |
+| d      | decimal data type   |
+| kb     | kilobyte multiplier |
+| mb     | megabyte multiplier |
+| gb     | gigabyte multiplier |
+| tb     | terabyte multiplier |
+| pb     | petabyte multiplier |
+
+There are two kinds of real literal: double and decimal. These are indicated by
+the absence or presence, respectively, of decimal-type suffix. PowerShell does
+not support a literal representation of a `[float]` value. A double real
+literal has type `[double]`. A decimal real literal has type `[decimal]`.
+Trailing zeros in the fraction part of a decimal real literal are significant.
+
+If the value of exponent-part's digits in a `[double]` real literal is less
+than the minimum supported, the value of that `[double]` real literal is 0. If
+the value of exponent-part's digits in a `[decimal]` real literal is less than
+the minimum supported, that literal is ill formed. If the value of
+exponent-part's digits in a `[double]` or `[decimal]` real literal is greater
+than the maximum supported, that literal is ill formed.
+
+> [!NOTE]
+> The syntax permits what a double real literal to have a long-type suffix.
+> PowerShell treats this as an integer literal whose value is represented by
+> type `[long]`. This feature has been retained for backwards compatibility
+> with earlier versions of PowerShell. However, programmers are discouraged
+> from using integer literals of this form as they can easily obscure the
+> literal's actual value. For example, `1.2L` has value 1, `1.2345e1L` has
+> value 12, and `1.2345e-5L` has value 0, none of which is immediately obvious.
+
+## Numeric multipliers
+
+For convenience, integer and real literals can contain a numeric multiplier,
+which indicates one of a set of commonly used powers of 10. The numeric
+multiplier can be written in any combination of upper or lowercase letters.
+
+The multiplier suffixes can be used in combination with the `u`, `ul`, and `l`
+type suffixes.
+
+### Multiplier examples
+
+```
+PS> 1kb
+1024
+
+PS> 1.30Dmb
+1363148.80
+
+PS> 0x10Gb
+17179869184
+
+PS> 1.4e23tb
+1.5393162788864E+35
+
+PS> 0x12Lpb
+20266198323167232
+```
+
+## Numeric type accelerators
+
+To pair with the new suffixes that closely reflect C\# or F\# literal suffixes,
+I've also added a couple additional type accelerators that alias existing ones:
```

Comment:
This sentence needs to be re-written to remove the **I've also added**.

Reply:
Fixed